src/admin-views/notices/upsell/icon.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Upsell Template
+ * The base template for TEC Upsell notices.
+ *
+ * Override this template in your own theme by creating a file at [your-theme]/tribe/upsell/upsell.php
+ *
+ * @since TBD
+ *
+ * @var string        $icon_url  URL to icon.
+ * 
+ * @version TBD
```

Comment:
Let's update the description. The admin views can't be overridden. Let's try to group the Since and version and then leave the var afterward. (the same for the other template.)